### ISSUE TITLE
PDF fail some accessibility checks

### ISSUE THREAD

**Issue by MrDeshaies** [canada-ca/aia-eia-js#609]
The generated PDF fail some accessibility checks. Attached are the reports from FoxIt. The primary issue they need the proper tagging structure.
[foxit pdf accessibility report.pdf](https://github.com/canada-ca/aia-eia-js/files/6063474/foxit.pdf.accessibility.report.pdf)

(reported by Andrew Wan, ESDC. Thanks!)

**Comment by MrDeshaies** [comment]
This one is not straightforward. Pdf is generated with a chain of libraries: html-to-pdfmake, pdfmake, and then pdfkit. It seems PDFkit has fairly good [support for accessibility](https://pdfkit.org/docs/accessibility.html), but it requires the code to be written a certain way to provide metadata and properly tag the structure of the document.

Unfortunately, pdfmake lacks support for accessing the accessibility features of pdfkit; it does not transfer certain required metadata (`lang, displayTitle, tagged`, which need to be set), and it offers only rudimentary support for tagging the doc sections (basic table of content or `toc` support).

That leaves us with two options... 1) enhance html-to-pdfmake and pdfmake to properly handle accessibility (non trivial), or 2) recommend users use a PDF editor after generation from AIA tool to complete the tagging and fix accessibility issues (which only takes 2 minutes per PDF with certain a certain editor common in GoC). I'll go with 2 in the short term, until 1 can be implemented.

**Comment by MrDeshaies** [comment]
In case I forget or someone asks: Chrome's Print / Save as PDF does not create accessible PDFs. I tried, and Foxit finds more accessibility issues than with the PDFs the AIA generates.

**Comment by grahhham** [comment]
Hi, our team is facing this issue quite alot recently, where we have been having trouble making the PDFs accessible using tagging in Foxit. Has there been any development towards implementing pdfKit?
